lib/Avatar/AvatarWithPopover.js

```diff
@@ -14,26 +15,48 @@ import { Avatar, Icon } from '../';
  *   ... popover content
  * </AvatarWithPopover>
  */
-const AvatarWithPopover = props => {
-  const popover = (
-    <Popover id={props.email} className={props.popoverClass}>
-      {props.children}
-    </Popover>
-  );
-
-  return (
-    <OverlayTrigger
-      trigger={props.triggerEvent}
-      placement={props.overlayPosition}
-      overlay={popover}
-    >
-      <button className="button button--icon-only avatar__wrapper">
-        <Avatar {...props} className="avatar--with-toggle" />
-        {props.caret && <Icon name="caret" className="avatar__caret" />}
-      </button>
-    </OverlayTrigger>
-  );
-};
+class AvatarWithPopover extends Component {
+  state = {
+    showing: false
+  };
+
+  render() {
+    const popover = (
+      <Popover id={this.props.email} className={this.props.popoverClass}>
+        {this.props.children}
+      </Popover>
+    );
+
+    const onEnter = () => {
+      this.setState({ showing: true });
+    };
+    const onExited = () => {
+      this.setState({ showing: false });
+    };
+
+    const activeClass = cx({
+      'is-active': this.state.showing
```

Comment:
What styling is applied for `is-active`?

Reply:
Adds highlighting for the caret icon